resource/subfunctions/resource_class.md

```diff
@@ -0,0 +1,323 @@
+```
+
+###### Security Result Dictionary as JSON
+```json
+{
+  "securityResult":{
+    "resource":{
+      "name":"SHELCITY",
+      "class":"CDT",
+      "operation":"set",
+      "requestId":"ResourceRequest",
+      "info":[
+        "Definition exists. Add command skipped due  to precheck option"
+      ],
+      "commands":[
+        {
+          "safReturnCode":0,
+          "returnCode":0,
+          "reasonCode":0,
+          "image":"RALTER  CDT             (SHELCITY)  CDTINFO  (FIRST       (ALPHA NUMERIC) OTHER       (ALPHA) PROFILESALLO(NO) DEFAULTRC   (4) DEFAULTUACC (READ))",
+          "messages":[
+            "IRR52199I Warning: Class name SHELCITY does not contain a national character nor a number.",
+            "ICH11009I RACLISTED PROFILES FOR CDT WILL NOT REFLECT THE UPDATE(S) UNTIL A SETROPTS REFRESH IS ISSUED."
+          ]
+        }
+      ]
+    },
+    "returnCode":0,
+    "reasonCode":0
+  }
+}
+```
+
+## `ResourceAdmin.extract_resource_class()`
+
+```python
+  def extract_resource_class( self, class_name: str ) -> Union[dict, bytes]:
+```
+
+#### 📄 Description
+
+Extract an existing **Resource Class** profile in the **CDT** class.
+
+#### 📥 Parameters
+* `class_name`<br>
+  The name of the **Resource Class** profile being defined to the **CDT** class.
+
+#### 📤 Returns
+* `Union[dict, bytes]`<br>
+  Returns a **Trait dictionary** of the values of the traits extracted from the **CDTINFO** segment of the Resource Profile or **Security Request XML bytes** if the `ResourceAdmin.generate_requests_only` class attribute is set to `True`.
+
+#### ❌ Raises
+* `SecurityRequestError`<br>
+  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+
+#### 💻 Example
+
+###### Python REPL
+```python
+from pyracf import ResourceAdmin
+resource_admin = ResourceAdmin()
+resource_admin.extract_resource_class("SHELCITY")
+{'case': 'upper', 'defaultrc': 8, 'defaultuacc': None, 'first': 'alpha', 'genlist': 'disallowed', 'generic': 'allowed', 'group': '', 'keyqualifiers': 0, 'macprocessing': 'normal', 'maxlength': 246, 'maxlenx': 246, 'member': '', 'operations': None, 'other': ['alpha', 'numeric'], 'posit': 200, 'profilesallowed': 'yes', 'raclist': 'allowed', 'seclabelsrequired': None, 'signal': None}
+```
+
+###### Trait Dictionary as JSON
+```json
+{
+  "case":"upper",
+  "defaultrc":8,
+  "defaultuacc":None,
+  "first":"alpha",
+  "genlist":"disallowed",
+  "generic":"allowed",
+  "group":"",
+  "keyqualifiers":0,
+  "macprocessing":"normal",
+  "maxlength":246,
+  "maxlenx":246,
+  "member":"",
+  "operations":None,
+  "other":[
+    "alpha",
+    "numeric"
+  ],
+  "posit":200,
+  "profilesallowed":"yes",
+  "raclist":"allowed",
+  "seclabelsrequired":None,
+  "signal":None
+}
+```

```

Comment:
Change `None` to `null` and add space in between `:` and values.

Reply:
Changed all None to null in all JSON dictionaries. (and fixed spacing)